Sources/App/Controllers/API/Package+Badge.swift

```diff
@@ -102,7 +102,7 @@ extension Package {
     
     static private func loadSVGLogo() -> String? {
         let pathToFile = Current.fileManager.workingDirectory()
-            .appending("Resources/SVGs/logo-simple.svg")
+            .appending("Public/images/logo-tiny.svg")
```

Comment:
Looks like we use `SiteURL.images("logo-small.png")` for the favicon - I wonder if the same syntax could be used here?

Reply:
SiteURL.images is going to generate a relative (or absolute, I guess!) URL, and don't we need a path to the file on disk here?

Reply:
You're entirely correct, yes. Ignore me 😄